<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestao_estoque</class>
 <widget class="QDialog" name="gestao_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_Novoproduto">
      <attribute name="title">
       <string>Novo produto</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Codigo do Produto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_codigodoproduto"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Descriçao do Produto </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_descricao"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Valor de compra</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_valorcompra"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Valor de venda</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_valordevenda"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Fornecedor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_fornecedor"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Qtde Estoque</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_qtdestoque"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_gravraproduto">
         <property name="text">
          <string>GRAVAR</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_novoproduto">
           <property name="text">
            <string>NOVO</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_gestaoproduto">
      <attribute name="title">
       <string>Gestão Produto</string>
      </attribute>
      <widget class="QTableWidget" name="tw_ge_produtos">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>100</y>
         <width>291</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Filtrar por:</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>122</width>
          <height>48</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QRadioButton" name="rb_ge_codigo">
           <property name="text">
            <string>Código Produto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_ge_des">
           <property name="text">
            <string>Descrição Produto</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QLineEdit" name="txt_ge_filtrar">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Filtrar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ge_filtrar">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>20</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>321</width>
         <height>351</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Codigo do Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_codigoproduto"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_produto"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Quantidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_quantidade"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Valor de Compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_valorcompra"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Valor de Venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_valorvenda"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_fornecedor"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_ge_gravar">
            <property name="text">
             <string>Gravar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ge_excluir">
            <property name="text">
             <string>Excluir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
